CMakeLists.txt

```diff
@@ -37,6 +37,9 @@ if (WIN32)
   # Use a newer version of CMake on Windows, so we are NEW for
   # https://cmake.org/cmake/help/latest/policy/CMP0149.html
   cmake_minimum_required(VERSION 3.27 FATAL_ERROR)
+
+  # set boost compiler to avoid errors with boost log
+  set(Boost_COMPILER "vc143")
```

Comment:
This should't be necessary: the boost infra should be making decisions based on the compiler actually in use.